OpenShift/helm/nexus-iq/values.yaml

```diff
@@ -0,0 +1,65 @@
+# Default values for iqserver.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+iq:
+  name: nxiq
+  applicationPort: 8070
+  adminPort: 8071
+  storageCapacity: 1Gi
+  memory: 1Gi
+  # base 64 encoded license file with no line breaks
+  licenseSecret: ""
+  # configYaml is the full text of the config.yml file that will be passed to IQ Server
```

Comment:
So is this now a nice yaml tree grafted into this yaml tree and not a string of yaml? Nice!